backend/oasst_backend/api/v1/utils.py

```diff
@@ -39,3 +39,7 @@ def prepare_tree(tree: list[Message], tree_id: UUID) -> protocol.MessageTree:
         tree_messages.append(prepare_message(message))
 
     return protocol.MessageTree(id=tree_id, messages=tree_messages)
+
+
+def prepare_user(u: User) -> protocol.User:
```

Comment:
I don't think this is well-named, I'd go with something descriptive like `db_user_to_protocol_user` or you could add a `@staticmethod` to the protocol user class like `from_db_user` to construct one from a db user. Or a `to_protocol_user` method on the db user class that transforms the db user into a protocol user. I think I'd go for the last option, since the second would create a back-dependency from shared to backend, and doesn't result in a separate extra lonely function like here.

Reply:
fixed